packages/console/src/pages/EnterpriseSsoDetails/Connection/index.tsx

```diff
@@ -29,6 +31,71 @@ type Props<T extends SsoProviderName> = {
   onUpdated: (data: SsoConnectorWithProviderConfigWithGeneric<T>) => void;
 };
 
+const isOidcFormData = (
+  formData: GuideFormType<SsoProviderName>
+): formData is GuideFormType<SsoProviderName.OIDC> => {
+  const keys = Object.keys(formData);
+  return (
+    keys.includes('clientId') ||
+    keys.includes('clientSecret') ||
+    keys.includes('issuer') ||
+    keys.includes('scope')
+  );
+};
+
+const isOidcConfig = (
+  config: SsoConnectorConfig<SsoProviderName>
+): config is SsoConnectorConfig<SsoProviderName.OIDC> => {
+  const keys = Object.keys(config);
+  return (
+    keys.includes('clientId') ||
+    keys.includes('clientSecret') ||
+    keys.includes('issuer') ||
+    keys.includes('scope')
+  );
+};
```

Comment:
They are almost identicle, isn't it strange?

Reply:
refactored and drop this utils methods.